docs/src/main/sphinx/installation/kubernetes.rst

```diff
@@ -0,0 +1,196 @@
+       pod/my-trino-cluster-coordinator-bfb74c98d-rnrxd   1/1     Running   0          161m
+       pod/my-trino-cluster-worker-76f6bf54d6-hvl8n       1/1     Running   0          161m
+       pod/my-trino-cluster-worker-76f6bf54d6-tcqgb       1/1     Running   0          161m
+
+       NAME                       TYPE        CLUSTER-IP    EXTERNAL-IP   PORT(S)    AGE
+       service/my-trino-cluster   ClusterIP   10.96.25.35   <none>        8080/TCP   161m
+
+       NAME                                           READY   UP-TO-DATE   AVAILABLE   AGE
+       deployment.apps/my-trino-cluster-coordinator   1/1     1            1           161m
+       deployment.apps/my-trino-cluster-worker        2/2     2            2           161m
+
+       NAME                                                     DESIRED   CURRENT   READY   AGE
+       replicaset.apps/my-trino-cluster-coordinator-bfb74c98d   1         1         1       161m
+       replicaset.apps/my-trino-cluster-worker-76f6bf54d6       2         2         2       161m
+
+Executing queries
+-----------------
+
+The pods running the Trino containers are all running on a private network
+internal to Kubernetes. In order to access them, specifically the coordinator,
+you will need to create a tunnel to the exposed service and your computer. You
+can do this by running the commands generated upon installation.
+
+#. Store the coordinator pod name in a shell variable called ``POD_NAME``.
+
+   .. code-block:: text
+
+       export POD_NAME=$(kubectl get pods --namespace default -l "app=trino,release=my-trino-cluster,component=coordinator" -o jsonpath="{.items[0].metadata.name}")
+
+#. Create the tunnel from the service to the client. After you run this you can
+   connect to the Trino coordinator at ``http://localhost:8080``.
+
+   .. code-block:: text
+
+       kubectl port-forward $POD_NAME 8080:8080
+
+#. To connect to Trino, you can use the :doc:`/client/cli` or your favorite
+   client that supports using a :doc:`/client/jdbc`.
+   :ref:`Install the command-line interface <cli-installation>`, and connect to
+   Trino.
+
+   .. code-block:: text
+
+       ./trino --server http://localhost:8080
+
+#. Using the generated benchmark data in the ``tpch`` catalog, type and execute
+   a query on the ``nation`` table using the scale factor schema, ``sf1``. Try
+   other commands to explore this data set:
+
+   .. code-block:: text
+
+       $ trino> select count(*) from tpch.sf1.nation;
+        _col0
+       -------
+         25
+       (1 row)
+
+       Query 20181105_001601_00002_e6r6y, FINISHED, 1 node
+       Splits: 21 total, 21 done (100.00%)
+       0:06 [25 rows, 0B] [4 rows/s, 0B/s]
+
+#. Once you are done with your exploration, enter the ``quit`` command.
+
+
+Configuring Helm for Trino
+--------------------------
+
+The :doc:`container image </installation/containers>` already contains a
+default configuration to get started, and some catalogs to allow you to explore
```

Comment:
you probably need to explain somewhere that the chart uses the container image..